packages/components/addon/components/hds/disclosure/index.js

```diff
@@ -18,7 +18,11 @@ export default class HdsDisclosureComponent extends Component {
     if (!this.toggleRef) {
       this.toggleRef = event.currentTarget;
     }
-    this.isOpen = !this.isOpen;
+    if (this.isOpen) {
```

Comment:
please add a comment here to explain why it is necessary to do this (otherwise in the future someone - likely me - will refactor this code, thinking that this is an "error") ;)

(and if possible, add a link to this issue so it's easy to find more details about the issue)

Reply:
👍